composer.json

```diff
@@ -23,8 +23,8 @@
     "require": {
         "php": "^5.6.0||^7.0.0",
         "friendsofsymfony/http-cache": "^2.0.0",
```

Comment:
here you need `^2.1.x-dev` for the time being

Reply:
ok, I will change that too